### PR TITLE
projects: adrv9009/ad9371: Add some sample profiles for test purposes

### PR DESCRIPTION
**Pull request by dbogdan**
The profiles were generated with MATLAB Filter Wizard / Profile Generator for ADRV9009 and AD9371/AD9375 Evaluation Software with GUI.

Signed-off-by: Dragos Bogdan <dragos.bogdan@analog.com>

### CODE REVIEW

File: projects/ad9371/profiles/tx_bw100_orx_bw100_rx_bw100/myk_ad9528init.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/* AD9528 data structure initialization file */
+#include <stdint.h>
+#include "common.h"
+#include "t_ad9528.h"
+
+static spiSettings_t clockSpiSettings = {
+	1, //chip select Index
```
Comment (mirceacaprioru) [inline projects/ad9371/profiles/tx_bw100_orx_bw100_rx_bw100/myk_ad9528init.c]: i know there was a discussion about designated initializer. Should we start using it ?

Comment (dbogdan) [reply projects/ad9371/profiles/tx_bw100_orx_bw100_rx_bw100/myk_ad9528init.c]: These files are generated by TES GUI, so these kind of changes should be implemented there.

**Comment by dbogdan** [comment]
The pull request was renamed since profiles for ad9371 were also added.

File: projects/ad9371/profiles/tx_bw100_orx_bw100_rx_bw100/myk_ad9528init.c (hunk 1 above)
Comment (mirceacaprioru) [reply projects/ad9371/profiles/tx_bw100_orx_bw100_rx_bw100/myk_ad9528init.c]: ack.

**Pull request merged**